cdap-gateway/src/main/java/io/cdap/cdap/gateway/router/RouterPathLookup.java

```diff
@@ -59,11 +59,13 @@ private enum AllowedMethod {
   public static final RouteDestination RUNTIME = new RouteDestination(Constants.Service.RUNTIME);
   public static final RouteDestination SUPPORT_BUNDLE_SERVICE =
       new RouteDestination(Constants.Service.SUPPORT_BUNDLE_SERVICE);
+  public static final RouteDestination SYSTEM_WORKER =
```

Comment:
I think this is unused now as we have removed the system worker handlers

Reply:
Thanks for catching! Fixed.